atomium/includes/common.inc

```diff
@@ -318,7 +318,7 @@ function atomium_get_component_definitions() {
         }
 
         // Disable the preview if we explicitely set the key disable to TRUE.
-        if (TRUE == $definition['disable']) {
+        if (TRUE === $definition['disable']) {
```

Comment:
This "code style change" is not really neutral to behavior.
The value in `$definition['disable']` could be coming from a function `THEMENAME_atomium_definition_COMPONENTNAME()`, which is out of our control, and could set it to something like '1' or numeric 1 or 'true' or whatever, which previously was interpreted as TRUE.

At least we know due to the array + above that the array key is set.

The behavior change can be ok, I would personally put in a dedicated commit with clear message. But up to you.